cluster/up.sh

```diff
@@ -35,9 +35,8 @@ fi
 if [[ "$KUBEVIRT_PROVIDER" =~ k8s- ]]; then
     echo 'Installing Open vSwitch and NetworkManager 1.34 on nodes'
     for node in $(./cluster/kubectl.sh get nodes --no-headers | awk '{print $1}'); do
-        ./cluster/cli.sh ssh ${node} -- sudo yum config-manager --set-enabled powertools
-        ./cluster/cli.sh ssh ${node} -- sudo yum install -y epel-release centos-release-openstack-wallaby epel-release centos-release-openstack-train https://rdoproject.org/repos/rdo-release.rpm
-        ./cluster/cli.sh ssh ${node} -- sudo yum install -y openvswitch libibverbs NetworkManager-1.34.0 NetworkManager-ovs-1.34.0
+        ./cluster/cli.sh ssh ${node} -- sudo dnf install centos-release-nfv-openvswitch -y
```

Comment:
can it we oneliner install ?
or at least move all the openvswitch to one separate line and the rest on the 2nd ?

please move the -y to the beginning as the other

Reply:
Have to be in two lines since we are installing a repository, I will move the `-y` but will keep installation of RPM at one line so we don't take more time to update repo database again.